err_stash.py

```diff
@@ -330,14 +488,31 @@ def stash_token(self, msg, args):
             return 'Stash plugin not configured, contact an admin.'
         if not args:
             if settings['token']:
-                return "You API Token is: `{}` (user: {})".format(settings['token'], user)
+                return "Your API Token is: `{}` (user: {})".format(settings['token'], user)
             else:
                 return NO_TOKEN_MSG.format(stash_url=self.config['STASH_URL'])
         else:
             settings['token'] = args[0]
             self.save_user_settings(user, settings)
             return "Token saved."
 
+    @botcmd(split_args_with=None)
+    def github_token(self, msg, args):
+        """Set or get Github token"""
+        user = msg.frm.nick
```

Comment:
I think that the user rocketchat nick is not necessarily the save login on github.

Reply:
That `user` is only used to load/save the RocketChat user `settings` dict, then `settings['github_token']` is used.